submit.go

```diff
@@ -224,68 +257,86 @@ func ParsArgs() *JobSubmit {
 	return jobSubmit
 }
 
-// GetStatus get status of the job from the API
-func (c *Client) GetStatus(projectID string, jobID string) *JobStatus {
-
-	job := &JobStatus{}
-	path := fmt.Sprintf(DataProcessingStatus, url.QueryEscape(projectID), url.QueryEscape(jobID))
-	if err := c.OVH.Get(path, job); err != nil {
-		log.Fatalf("Unable to retrieve status for job: %s", err)
+//poll Status
+func Loop(c *Client, job *JobStatus) {
+	sigs := make(chan os.Signal, 2)
+	signal.Notify(sigs, os.Interrupt, syscall.SIGTERM)
+	var err error
+statusLoop:
+	for {
+		select {
+		case <-sigs:
+			var s string
+			fmt.Printf("Do you want to kill the Job (y/N): ")
+			_, err := fmt.Scan(&s)
+			if err != nil {
+				log.Printf("Job not killed")
+			}
+
+			s = strings.TrimSpace(s)
+			s = strings.ToLower(s)
+
+			if s == "y" || s == "yes" {
+				if err := c.Kill(args.ProjectID, c.JobID); err != nil {
+					log.Printf("Job not killed: %d", err)
+				}
+				log.Printf("Job killed")
+			} else {
+				log.Printf("Job not killed")
+			}
+			return
+		case <-time.After(LoopWaitSecond * time.Second):
+			job, err = c.GetStatus(args.ProjectID, job.ID)
+			if err != nil {
+				log.Printf("Unable to retrieve status for job: %s", err)
+			}
+			switch job.Status {
+			case JobStatusUNKNOWN, JobStatusSUBMITTED, JobStatusPENDING:
+				log.Printf("Job is %s", job.Status)
+
+			case JobStatusCANCELLING, JobStatusTERMINATED, JobStatusFAILED, JobStatusCOMPLETED:
+				break statusLoop
+
+			case JobStatusRUNNING:
+				if jobLog, err := c.GetLogLast(args.ProjectID, job.ID); err == nil {
+					c.lastPrintLog = PrintLog(jobLog.Logs)
+				} else {
+					log.Printf("Unable fetch job log: %s", err)
+				}
+			}
```

Comment:
Can you add default case in case we add other statuses at one point and "forget" to update this code?